.github/workflows/publish.yml

```diff
@@ -1,55 +1,55 @@
-name: Publish stable packages
+name: Publish packages
+
+# single workflow for both preview and stable as we can only specify a single
```

Comment:
Where do we specify this ?

Reply:
In the package settings on npm. I'll write some docs so maintainers know how to publish and configure new packages.